bin/ci.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -eux
```

Comment:
`-x` makes bash log every command before executing it. So that we can see exactly which steps produced which output (or failed) on github.